### PR TITLE
Change pubsub get_message API timeout to seconds.

### PR DESCRIPTION
**Pull request by liuh-80**
Change pubsub get_message API timeout to seconds.

#### Why I did it
In redis-py library the pubsub timeout is seconds, to keep compatibility, change pubsub get_message API timeout to seconds.
The only code using pubsub get_message API with timeout is frrcfgd, change this API will not break any other code in SONiC.

##### Work item tracking
- Microsoft ADO: 27002941

#### How I did it
Change pubsub get_message API timeout to seconds.

#### How to verify it
Pass all UT.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

#### Description for the changelog
Change pubsub get_message API timeout to seconds.

### CODE REVIEW

File: common/pubsub.cpp [hunk 1]
```diff
@@ -77,12 +77,12 @@ bool PubSub::hasCachedData()
     return m_keyspace_event_buffer.size() > 1;
 }
 
-map<string, string> PubSub::get_message(double timeout, bool interrupt_on_signal)
+map<string, string> PubSub::get_message(double seconds, bool interrupt_on_signal)
```
Comment (qiluo-msft) [inline common/pubsub.cpp:80]: Let's change back to `timeout`. `seconds` will lose the semantics.

Comment (liuh-80) [reply common/pubsub.cpp]: Fixed

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**